styles/components/_listingsGrid.scss

```diff
@@ -2,12 +2,12 @@
    the store and search results views. Make sure to check both when making changes to this file.
  */
 
-$crytoListViewTradeFromWidth: 57px;
-$crytoListViewTradeToWidth: 104px;
-$crytoListViewTradeArrowWidth: 48px;
-$crytoListViewVendorWidth: 185px;
-$crytoListViewPriceWidth: 125px;
-$crytoListViewInventoryMaxWidth: 138px;
+$crytoListViewTradeFromWidth: 9%;
```

Comment:
hmmm... a lot of tinkering was put in place to make sure different permutations of the variable length content would all look nice. If you're implementation is bigger, we're ok. If it's smaller, maybe not.